src/IntegrationUtils.ts

```diff
@@ -98,6 +117,11 @@ function createNewIntegration(integrationFileUri: vscode.Uri, configmap? : strin
 		if (secret && secret.trim().length > 0) {
 			commandString += ` --secret=${secret}`;
 		}
+		if (resource) {
+			// assuming that the file is in a path relative to the integration file
```

Comment:
I agree that this will not be a common use case. Perhaps we move it to being a command only available in the command palette? As for file path, every example I have seen has been relative. We can try an absolute path, but I'm not sure if it will work or not.

Reply:
It looks like this will work - `kamel run --resource="/home/bfitzpat/Documents/camelk-examples/resources-data.txt" "resources-route.groovy"`

And "path.normalize" based on the docs will work on the Windows side as well.